.devcontainer/devcontainer.json

```diff
@@ -18,7 +18,8 @@
     "--volume=${localEnv:HOME}/.comma:/root/.comma",
     "--volume=/tmp/comma_download_cache:/tmp/comma_download_cache",
     "--volume=/tmp/devcontainer_scons_cache:/tmp/scons_cache",
-    "--shm-size=1G"
+    "--shm-size=1G",
+    "--add-host=host.docker.internal:host-gateway" // required to use host.docker.internal on linux
```

Comment:
@fredyshox can you check if this breaks anything on mac? I don't see a way to gate this flag for linux only

Reply:
seems to work fine on windows, so I'd assume it will work find on mac too